catalogd/internal/controllers/core/clustercatalog_controller.go

```diff
@@ -229,6 +230,7 @@ func (r *ClusterCatalogReconciler) reconcile(ctx context.Context, catalog *catal
 
 	unpackResult, err := r.Unpacker.Unpack(ctx, catalog)
 	if err != nil {
+		httputil.LogUnverifiedCertificate(err, ctrl.Log.WithName("catalog-unpacker"))
```

Comment:
Should we do this same thing where operator-controller is calling unpack for bundles?

Reply:
I was thinking the same last night after I signed off...